File: Level1/14.서울에서김서방찾기/14.서울에서김서방찾기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

string solution(vector<string> seoul) {
    string answer = "";
    int result = find(seoul.begin(), seoul.end(), "Kim") - seoul.begin();
    answer = "김서방은 "+to_string(result)+"에 있다";
    return answer;
}

File: Level1/14.서울에서김서방찾기/14.서울에서김서방찾기.java
import java.lang.StringBuilder;
class Solution {
    public String solution(String[] seoul) {
        StringBuilder answerSB = new StringBuilder();
        for(int i = 0 ; i<seoul.length ; i++){
            if(seoul[i].equals("Kim")){
                answerSB.append("김서방은 ");
                answerSB.append(Integer.toString(i));
                answerSB.append("에 있다");
                break;
            }
        }
        String answer = answerSB.toString();
        return answer;
    }
}